repositories.yaml

```diff
@@ -1507,6 +1507,30 @@ applications:
       - v1_name: glean-dictionary
         app_id: glean-dictionary
 
+  - app_name: mdn_fred
+    canonical_app_name: Mozilla Developer Network
+    app_description: |
+      MDN Web Docs is an open-source, collaborative project documenting Web
+      platform technologies, including CSS, HTML, JavaScript, and Web APIs.
+      MDN also provides an extensive set of learning resources for beginning
+      developers and students.
```

Comment:
This description shows up on the dictionary (see https://dictionary.telemetry.mozilla.org/)
Can we do add something to better differentiate the two apps? They otherwise have the exact same display name and description.

Reply:
Fixed in 421aa72.